src/snowflake/snowpark/modin/plugin/_internal/index.py

```diff
@@ -1688,8 +1745,7 @@ def tolist(self) -> list:
         >>> idx.to_list()
         [1, 2, 3]
         """
-        # TODO: SNOW-1458117 implement tolist
-        return self.to_pandas().tolist()
+        return self.values.tolist()
```

Comment:
the original implementation seems fine to me, what is the reason to update this to values?

Reply:
It's one query either way, but I wanted to use existing methods rather than calling to_pandas()